lib/assets/stylesheets/modules/_search.scss

```diff
@@ -19,6 +12,48 @@
   }
 }
 
+.js .search__label {
+  @extend .govuk-visually-hidden;
+}
+
+.search__form {
+  position: relative;
+  width: 100%;
+}
+
+.search__input {
+  width: 230px;
+  width: calc(100% - #{$input-size});
+}
+
+.search__button {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  border: 0;
+  background-color: govuk-colour('blue');
+  color: #fff;
+  cursor: pointer;
+  border-radius: 0;
```

Comment:
Is border-radius needed? Is that for a specific browser?

Reply:
Nope, I cross-browser tested and it doesn't seem like it's needed after all. I think I had just copied it over from the GOVUK search button styles.